src/drive/web/modules/views/OnlyOffice/styles.styl

```diff
@@ -0,0 +1,4 @@
+.title {
+    width calc(100% - 1rem) !important // to override MUI style
```

Comment:
which one? Is it wanted? Should it be handled is UI?

Reply:
le `width` de `DialogTitle`. Comme on n'utilise pas le `makeStyle` de MUI, si on veut override un style présent dans un composant MUI on est obligé de passer par les `!important`.

Est-ce que du coup ici utiliser `makeStyle` fait plus de sens ?

Reply:
> Est-ce que du coup ici utiliser makeStyle fait plus de sens ?

Oui je pense. Vraiment pour la surcharge, c'est plus simple de makeStyle()